### PR TITLE
188 rename supervisors to management

### PR DESCRIPTION
**Pull request by OttoChen312**
## Description
Made some changes to the strings in files to have better consistency of capitalization across the website, and made changes to the people page to change supervisor to management and filtered them by adding only people with management position

## Updates

[Please enter a brief summary of updates to this PR during code review - if any]

## Checklist

- [ ] Ran `npm run lint:fix` to format code
- [ ] Requested at least one reviewer
- [ ] Connected PR to Trello ticket (steps below)
- [ ] Addressed code review comments
- [ ] Gained at least one approval for your PR

## Connecting your PR to the corresponding ticket:
- Click on the "GitHub" button found on the right of your Trello ticket
- Choose "Attach PR" from the dropdown list
- Link your GitHub account (if you haven't done so already)
- Navigate to`VCL-content-platform` and choose your PR from the dropdown.

### CODE REVIEW

**Comment by OttoChen312** [comment]
[Consistency in capitalization across website](https://trello.com/c/J5QBZlc3/190-consistency-in-capitalization-across-website)

**Comment by OttoChen312** [comment]
[Rename supervisors to management](https://trello.com/c/32c2llYS/188-rename-supervisors-to-management)

**Comment by ElisaHu1** [comment]
@OttoChen312 there is a conflict file, can you update that?

**Comment by ElisaHu1** [comment]
@OttoChen312 you failed the linting lol

**Comment by OttoChen312** [comment]
Oh sorry I changed it on GitHub, I should have changed on Intelliji, I will implement the changes through Intelliji.

**Comment by OttoChen312** [comment]
I will review the conflicts on Intelliji by tonight sorry about that

**Review by ElisaHu1 — APPROVED** [review]
looks good to me! waiting for @ryanyae to review it

File: src/pages/People/People.tsx [hunk 1]
```diff
@@ -118,18 +118,25 @@ const People = () => {
...
               currentList.filter((item) => {
-                return (
-                  item.position == currentProject || item.project == currentProject
-                );
-
-              }).length > 0 ? (
-                currentList
-                  .filter((item) => {
+                  if (currentProject == "Management") {
+                    return item.position === 'Lab Manager' || item.position === 'Lab Leader' || item.position === 'Assistant Lab Manager and Workshop Coordinator' || item.position === 'Workshop Coordinator';
```
Comment (ryanyae) [inline src/pages/People/People.tsx:122]: When comparing 2 strings with "===", the equality behaviour will be case-sensitive. This is very nit-picky, but I think it's better to have a non-case-sensitive implementation to ensure we are not filtering out typos.

File: src/pages/People/People.tsx [hunk 2]
```diff
@@ -118,18 +118,25 @@ const People = () => {
...
-                      item.project == currentProject
+                     item.position == currentProject || item.project == currentProject
                     );
-                  })
+                  }
+                }).length > 0 ? (
+                  currentList
+                    .filter((item) => {
+                      if (currentProject == "Management") {
+                        return item.position === 'Lab Manager' || item.position === 'Lab Leader' || item.position === 'Assistant Lab Manager and Workshop Coordinator' || item.position === 'Workshop Coordinator';
```
Comment (ryanyae) [inline src/pages/People/People.tsx:132]: Same thing here with case-sensitivity.

**Review by ryanyae — CHANGES_REQUESTED** [review]
Just address the 2 comments I left, besides that everything else looks good to me, good job Otto!

File: src/pages/People/People.tsx [hunk 3]
```diff
@@ -118,18 +118,25 @@ const People = () => {
...
               currentList.filter((item) => {
-                return (
-                  item.position == currentProject || item.project == currentProject
-                );
-
-              }).length > 0 ? (
-                currentList
-                  .filter((item) => {
+                  if (currentProject == "Management") {
+                    return item.position.toLowerCase() == 'Lab Manager'.toLowerCase() || item.position.toLowerCase() == 'Lab Leader'.toLowerCase() || item.position.toLowerCase() == 'Assistant Lab Manager and Workshop Coordinator'.toLowerCase() || item.position.toLowerCase() == 'Workshop Coordinator'.toLowerCase();
```
Comment (ryanyae) [inline src/pages/People/People.tsx:122]: Converting all these to lower case is actually redundant and not necessary, when using "==" its not case sensitive so "ADasd" will equal "adasd". So I think taking it out is for the best.

File: src/pages/People/People.tsx [hunk 4]
```diff
@@ -118,18 +118,25 @@ const People = () => {
...
-                      item.project == currentProject
+                     item.position == currentProject || item.project == currentProject
                     );
-                  })
+                  }
+                }).length > 0 ? (
+                  currentList
+                    .filter((item) => {
+                      if (currentProject == "Management") {
+                        return item.position.toLowerCase() == 'Lab Manager'.toLowerCase() || item.position.toLowerCase() == 'Lab Leader'.toLowerCase() || item.position.toLowerCase() == 'Assistant Lab Manager and Workshop Coordinator'.toLowerCase() || item.position.toLowerCase() == 'Workshop Coordinator'.toLowerCase();
```
Comment (ryanyae) [inline src/pages/People/People.tsx:132]: same thing here.

**Review by ryanyae — CHANGES_REQUESTED** [review]
Everything else looks great, just address my comments and we can merge this.

**Review by ryanyae — APPROVED** [review]

**Pull request merged**